src/components/grapher/GrapherDots.vue

```diff
@@ -1,24 +1,45 @@
 <template>
   <g class="grapher-dots--dots-container">
     <circle
-      v-for="dot in stuntSheetDots"
+      v-for="dot in stuntSheetDotsUnselected"
       :key="`${dot.x}-${dot.y}-dots--dot`"
       class="grapher-dots--dot"
       :cx="dot.xAtBeat(beat - 1)"
       :cy="dot.yAtBeat(beat - 1)"
       r="0.7"
       data-test="grapher-dots--dot"
     />
+    <circle
+      v-for="dot in stuntSheetDotsSelected"
+      :key="`${dot.x}-${dot.y}-dots--dot-selected`"
+      class="grapher-dots--dot-selected"
+      :cx="dot.xAtBeat(beat - 1)"
+      :cy="dot.yAtBeat(beat - 1)"
+      r="0.7"
+      data-test="grapher-dots--dot-selected"
+    />
```

Comment:
Nice! When we implement @adhamrait 's PR for grapher dot types, then maybe we can add a prop for "selected" that adds a border/changes the color!